### PR TITLE
Added locally hosted SLS2018 page.

### PR DESCRIPTION
**Pull request by feffemannen**

### CODE REVIEW

File: _events/2018-06-11-sls2018.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+LaTeX source files.
+
+### Conference dinner
+
+The dinner will be held at [Villa Belparc](http://www.villabelparc.se/)
+located in the beautiful park Slottskogen right by the [Botanical
+Gardens](http://www.botaniska.se/en).
+
+-   Dinner fee (incl. VAT). 700 SEK
+
```
Comment (greleigh) [inline _events/2018-06-11-sls2018.md:157]: remove blank line between bullets

File: _events/2018-06-11-sls2018.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
+---
+layout: page
+title: The Tenth Scandinavian Logic Symposium
+permalink: events/sls2018/
+---
+{% assign assets = "assets/events/18-sls" %}
+
+The Tenth Scandinavian Logic Symposium (SLS 2018) will be held at the University of Gothenburg, Sweden during June 11-13, 2018, under the auspices of the Scandinavian Logic Society. The previous two meetings of the SLS were held in Tampere, Finland (2014) and Roskilde, Denmark (2012).
```
Comment (greleigh) [inline _events/2018-06-11-sls2018.md:8]: past tense now: *was held*

**Review by greleigh — COMMENTED** [review]
Looks great. Just two minor comments (see below)

**Comment by feffemannen** [comment]
Thanks. Should be changed now.

**Pull request merged**